packages/react-searchbox/src/components/SearchComponent.js

```diff
@@ -100,12 +100,16 @@ class SearchComponent extends React.Component {
       maxPopularSuggestions,
       stopwords
     } = this.props;
-    let { value } = this.props;
+    let { value, categoryValue: category } = this.props;
     if (window && window.location && window.location.search) {
```

Comment:
@mohdashraf010897 This is not the right place to handle URL params. Check the implementation in `URLParamsProvider` and remove it from `SearchComponent`.

Reply:
isn't this an existing thing?

Reply:
Yes, added by you.